### PR TITLE
Oleg final checklist

### PR DESCRIPTION
**Pull request by orovinskiy**
Finished with the styling and auto populating and also made the checkbox move to their correct place when clicked.

### CODE REVIEW

File: controller/Controller.php [hunk 1]
```diff
@@ -62,4 +62,45 @@ function tutorsAjax()
...
+     * @author Oleg
+     */
+    function checklistAjax(){
+        $this->_db->updateYearData($_POST['column'],$_POST['value'],$_POST['year']);
+    }
+
+    /**
+     * View of all the required forms. Lets the user check of or uncheck a form if it has been
+     * completed
+     * @param $param int is the id of the user
```
Comment (kellerflint) [inline controller/Controller.php]: We've been doing @param datatype $varibable. Either is PEAR but it should be consistent

File: model/database.php [hunk 2]
```diff
@@ -103,7 +103,7 @@ function updateYearData($column, $value, $yearId)
     {
 
         // TODO validate $column against php array of column names used in db to prevent sql injection
-
+        echo 'Im reaching this';
```
Comment (kellerflint) [inline model/database.php]: remove test lines

File: model/database.php [hunk 3]
```diff
@@ -175,4 +175,25 @@ function testDatabase()
...
     }
 
+    /** This fetches all the information to be displayed for the tutors view.
+     * Only important data is chosen.
+     * @param int $year the year that is required
+     * @param int $userID the database id of the tutor
+     * @return array returns all the required checkboxes as well as the name
+     * @author oleg
+     */
+    function getTutorsChecklist($year, $userID){
```
Comment (kellerflint) [inline model/database.php:184]: I think this function is redundant, Dallas wrote one called getTutor that does the same thing.

File: model/db_schema.sql [hunk 4]
```diff
@@ -49,6 +49,35 @@ CREATE TABLE Year (
     FOREIGN KEY (user_id) REFERENCES Tutor (user_id) ON UPDATE CASCADE
 );
 
+/*
+Test Data
+@author Oleg
+*/
+
+INSERT INTO User Values (default,"Bobby@mail.com","bobTroll!!",0),(default,"Andy@mail.com","AndyHot!!",0),(default,"CoolGuy@mail.com","notCool!!",0),
```
Comment (kellerflint) [inline model/db_schema.sql:57]: im worried your test data is going to conflict with Dallas's test data. If you haven't already, please test on your workbench with both sets of data together to make sure it doesn't throw errors.

File: styles/checklist.css [hunk 5]
```diff
@@ -14,3 +14,12 @@ legend{
     padding-left: 5%;
 }
 
+input {
+    transform: scale(1.3);
+    -webkit-transform: scale(1.3);
+}
+
+.card{
```
Comment (kellerflint) [inline styles/checklist.css]: stylesheet elements go in alphabetical order

**Review by kellerflint — CHANGES_REQUESTED** [review]

File: model/database.php (hunk 2 above)
Comment (orovinskiy) [reply model/database.php]: Done

File: model/database.php (hunk 3 above)
Comment (orovinskiy) [reply model/database.php:184]: his gets extra information that I do not use. Is that fine?

File: model/db_schema.sql (hunk 4 above)
Comment (orovinskiy) [reply model/db_schema.sql:57]: works like a charm now

File: model/database.php (hunk 3 above)
Comment (dsloan92) [reply model/database.php:184]: yeah, may be redundant, but we can keep until we decide as a group if we wan't to keep or not.

File: model/db_schema.sql (hunk 4 above)
Comment (dsloan92) [reply model/db_schema.sql:57]: Thanks for adding to the test data

**Review by dsloan92 — APPROVED** [review]
looks good. Good commenting. Thanks for adding to the test data.

**Pull request merged**